### DISCUSSION TITLE
Ugly "General Math" menu

### DISCUSSION THREAD

**Discussion by Donatien-Alphonse** [wxMaxima-developers/wxmaxima#1606] [Q&A]
Hello friends,

My question concerns how to make the dockable "General Math" menu look decent. In the newest version of wxMaxima that I'm using on Linux Mint, all of the commands are run-together. See the following picture.

![bad](https://user-images.githubusercontent.com/93812232/140704086-cd8f2317-11a8-4e8e-80e4-5f514478f04f.jpg)

This wasn't the case on previous versions, where the menu is quite attractively laid out. See the next picture.

![good](https://user-images.githubusercontent.com/93812232/140704104-b7755d39-2ecd-4125-a930-2d218f37c988.jpg)

Any ideas on how to get this working decently again?

Many thanks,

Donatien-Alphonse

**Comment by daute** [comment]
It seems to be a change in wxWidgets. The current wxMaxima source compiled using wxWidgets 3.0.4 looks like the screeenshot below, using wxWidgets 3.1.5 it looks like the first screenshot.

**Comment by daute** [comment]
Thanks for your report. Should be fixed with commit 9ba8686.
